episodes/episodes.Rmd

```diff
@@ -356,7 +356,7 @@ A: part of the precipitate
 :::::::::::::::::::::::::
 :::::::::::::::::::::::::::::::::::::::::::::::
 
-To add more solutions, you close the first solution and add more text:
+To add more challenges, you close the first solution and add more text:
```

Comment:
I agree it is a wonky sentence, but it's not that we are adding more challenges, so what about this?

```suggestion
To add more content to the challenge, you close the first solution and add more text:
```